src/creepSetups/setups.ts

```diff
@@ -229,6 +229,11 @@ export const CombatSetups = {
 			sizeLimit: Infinity,
 		}),
 
+		distraction:  new CreepSetup(Roles.ranged, {
+			pattern  : [MOVE, MOVE, MOVE, RANGED_ATTACK, MOVE],
```

Comment:
I think you meant for this to have 5 move parts, right?
```suggestion
			pattern  : [MOVE, MOVE, MOVE, MOVE, RANGED_ATTACK, MOVE],
```

Reply:
Yeah, never updated this PR since mu is gone. It's got 5 on my branch. Thanks!